packages/@okta/vuepress-site/docs/guides/oie-upgrade-plan-embedded-upgrades/main/index.md

```diff
@@ -88,7 +88,7 @@ When you're embedding your authentication with our SDKs, consider all the steps
 
 ## Upgrade from using the Okta Authentication APIs
 
-With some solutions, there are more architectural evaluations from Architects/Engineering Leadership/Product teams that need to be made, planned, and executed to address technical debt or, in the case of API-based implementations, move away from a less supported implementation. More detailed information is coming soon on moving away from using Classic Authentication APIs to using the appropriate SDK.
+With some solutions, there are more architectural evaluations from Architects/Engineering Leadership/Product teams that need to be made, planned, and executed to address technical debt or, in the case of API-based implementations, move away from a less supported implementation. More detailed information is coming soon on moving away from using classic Authentication APIs to using the appropriate SDK.
```

Comment:
Would it be clearer to call these Classic Engine Authentication APIs? I don't think lowercase "classic" works here, it reads like a simple adjective (unless this is an API term I'm not familiar with?)

Reply:
Yes, good catch. Thanks